frontend/packages/model-serving-backport/extensions.ts

```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line no-restricted-syntax
+import { SupportedArea } from '@odh-dashboard/internal/concepts/areas/types';
+// eslint-disable-next-line no-restricted-syntax
+import { ProjectObjectType } from '@odh-dashboard/internal/concepts/design/utils';
+// eslint-disable-next-line no-restricted-syntax
+import { NamespaceApplicationCase } from '@odh-dashboard/internal/pages/projects/types';
+import type { ModelServingPlatformExtension } from '@odh-dashboard/model-serving/extension-points';
+
+const extensions: ModelServingPlatformExtension[] = [
+  {
+    type: 'model-serving.platform',
+    properties: {
+      id: 'model-mesh',
+      manage: {
+        namespaceApplicationCase: NamespaceApplicationCase.MODEL_MESH_PROMOTION,
+        enabledProjectMetadata: {
+          labels: {
+            'modelmesh-enabled': 'true',
+          },
+        },
+      },
+      enableCardText: {
+        title: 'Multi-model serving platform',
+        description:
+          'Multiple models can be deployed on one shared model server. Choose this option when you want to deploy a number of small or medium-sized models that can share the server resources..',
+        selectText: 'Select multi-model',
+        enabledText: 'Multi-model serving enabled',
+        objectType: ProjectObjectType.modelServer,
```

Comment:
```suggestion
        objectType: ProjectObjectType.multiModel,
```
One small nit is that the icon here doesn't match the existing one. Not sure if this is intentional or not.
Otherwise everything looks and is working great!